devtools/conda-envs/environment.yml

```diff
@@ -4,17 +4,18 @@ channels:
   - omnia
   - rdkit
   - samoturk
+  - schrodinger
 dependencies:
   - pytest
   - biopandas
   - jupyter
   - matplotlib
   - numpy
-  - pandas
-  - rdkit
+  - pandas= 2.1
+  - rdkit=2021.09
```

Comment:
@AndreaVolkamer the `rdkit` version here is not in sync with the KinFragLib repo. Not necessarily a problem but maybe for the future, those could be in sync?